app/jobs/track_agent_statuses_job.rb

```diff
@@ -12,6 +13,14 @@ def perform
     AgentStatusUpdater.new(now, log.last_success).update_statuses(current)
   end
 
+  def agent_has_eaten(data)
+    agent_id = data[:agent_id].to_i
+    luncheds = Rails.cache.fetch('lunched', force: true)
+    return if luncheds.nil?
```

Comment:
What does this do? Doesn't this mean that nothing is stored in the cache if it's nil?